src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3449,6 +3449,304 @@ def __repr__(self) -> str:
         return repr_str
 
 
+class Decode3DInputsAffineTransforms(TopdownAffine):
```

Comment:
Could you add unit test for this class?

Reply:
Sure, will do

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4046/commits/ae60b71090bdc8f7e00776055de3f2c7e9684831

Reply:
This transform has multiple functionality because it is hard to divide transforms from decoding KITTI annotations for 3d detection training. The annotations should use info from affine transforms. I created this unified transform to move resizing, cropping and flipping from __get_item__ in the Dataset. 

This transform inherits from TopdownAffine, because it uses 2 its main methods, "_get_warp_matrix" and " _get_warp_image"

Reply:
Thanks for the explanation. If so, I agree with your decision.

Reply:
Added more unit tests: https://github.com/openvinotoolkit/training_extensions/pull/4046/commits/23d767dcc23942efb027e24f3872010542530abb